github-approve-label-workflow/github-approve-label-workflow.py

```diff
@@ -0,0 +1,174 @@
+#
+# Do we have any open PR's that have label "Approval: done"
+# that are over 24 hours without any other comments?
+#
+# get a token.... https://github.com/settings/tokens/new -- just repo is fine
+# pop it in token.txt or you'll get a bad API limit
+#
+# note that we'd use pyGithub but we can't as it doesn't fully handle the timeline objects
+# as of Feb 2020
+#
+# mark@openssl.org Feb 2020
+#
+import requests
+import json
+from datetime import datetime, timezone
+from optparse import OptionParser
+
+api_url = "https://api.github.com/repos/openssl/openssl"
+
+
+def convertdate(date):
+    # python fromisoformat needs a TZ in hours/minutes
+    return datetime.fromisoformat(date.replace('Z', '+00:00'))
+
+
+# Get all the open pull requests, filtering by approval: done label
+#
+
+
```

Comment:
My python was a bit all over the place so I ran it through yapf which did that.  Do we have a preffered python style?

Reply:
Well we don't have a lot of python - so no, I don't think so. For perl/C I think we would tend to avoid extraneous blank lines like this - although again I'm not sure we have a formal rule on it. Consider my comment "optional" to address.